src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -5140,3 +5141,62 @@ class CreateExternalDataSourceStatementSegment(BaseSegment):
             ),
         ),
     )
+
+class CreateExternalTableStatementSegment(BaseSegment):
+    """A `CREATE EXTERNAL TABLE` statement
+
+    https://learn.microsoft.com/en-us/sql/t-sql/statements/create-external-table-transact-sql?view=sql-server-ver16&tabs=dedicated
+    """
+
+    type = "create_external_table_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        "EXTERNAL",
+        "TABLE",
+        Ref("ObjectReferenceSegment"),
+        Bracketed(
+            Delimited(
+                AnyNumberOf(Ref("ColumnDefinitionSegment")),
```

Comment:
You don't need this AnyNumberOf inside Delimited, because Delimited extends AnyNumberOf